website/addons/dryad/static/dryadNodeConfig.js

```diff
@@ -0,0 +1,508 @@
+'use strict';
+
+var ko = require('knockout');
+require('knockout.punches');
```

Comment:
The interface isn't working for me because `knockout.punches` is missing.  We took that out in v0.72.